cmd/lakefs/cmd/run.go

```diff
@@ -53,9 +53,7 @@ var runCmd = &cobra.Command{
 		// validate service names and turn on the right flags
 		dbParams := cfg.GetDatabaseParams()
 
-		if err := db.ValidateSchemaUpToDate(dbParams); errors.Is(err, db.ErrSchemaNotCompatible) {
```

Comment:
Committed by mistake?

Reply:
Need a failure to see this one works first